lib/prometheus-metrics.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+class PrometheusMetrics {
+  constructor (circuit) {
+    this.circuit = circuit;
+    // this.client = require('prom-client');
+    this.collect();
```

Comment:
Here, create an array of all the measurements we want to take.

```suggestion
    const openCounter = new client.Counter({
      name: 'circuit-open',
      help: 'A cumulative count of the times the circuit has opened'
    });

    const fireCounter = new client.Counter({
      name: 'circuit-open',
      help: 'A cumulative count of the times the circuit has fired'
    });
    this.counters = [ openCounter, fireCounter ]; // etc.
```